charts/operator/templates/crds/allocator.yaml

```diff
@@ -0,0 +1,281 @@
+{{ if .Values.global.operator.enabled }}
```

Comment:
we need crd: create: true something like that because most orgs dont have access to create crd on default they will ask their kubernetes admin or security team to apply them

Reply:
I can keep it under global like:

```
operator:
    enabled: true
    crd:
        create: true
```

Reply:
added the above

Reply:
globally not required we can add those under operator chart component